sse2neon.h

```diff
@@ -5118,6 +5118,19 @@ FORCE_INLINE __m128d _mm_cmplt_sd(__m128d a, __m128d b)
 #endif
 }
 
+// Compare packed double-precision (64-bit) floating-point elements in a and b
+// for not-equal, and store the results in dst.
+// https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_cmpneq_pd
+FORCE_INLINE __m128d _mm_cmpneq_pd(__m128d a, __m128d b)
+{
+    // get an all ones vector
+    __m128d all_ones = vreinterpretq_m128d_u64(vdupq_n_u64(UINT64_MAX));
+    __m128d res_eq = _mm_cmpeq_pd(a, b);
```

Comment:
Instead of composting with `_mm_cmpeq_pd`, you should implement in fewer instructions.